DependencyInjection/Compiler/DoctrineDBALLoggerPass.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace Doctrine\Bundle\DoctrineBundle\DependencyInjection\Compiler;
+
+use Doctrine\DBAL\Logging\SQLLogger;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Definition;
+use Symfony\Component\DependencyInjection\Exception\InvalidArgumentException;
+use Symfony\Component\DependencyInjection\Reference;
+
+final class DoctrineDBALLoggerPass implements CompilerPassInterface
+{
+    private const TAG_NAME = 'doctrine.dbal.logger';
+    private const LOGGER_INTERFACE = SQLLogger::class;
```

Comment:
I would remove this constant, and use `SQLLogger::class` directly in the `implementsInterface` call